c2corg_images/views.py

```diff
@@ -105,3 +105,15 @@ def publish(request):
     for key in resized_keys(filename):
         incoming_storage.move(key, active_storage)
     return {'success': True}
+
+
+@view_config(route_name='delete', renderer='json')
+def delete(request):
+    if request.POST['secret'] != os.environ['API_SECRET_KEY']:
+        raise HTTPForbidden('Bad secret key')
+    filenames = request.POST.getall('filenames')
+    for filename in filenames:
+        active_storage.delete(filename)
+        for key in resized_keys(filename):
+            active_storage.delete(key)
```

Comment:
In case `active_storage.delete(key)` fails for one of the keys, shouldn't we continue with the other keys and filenames?